packages/cli/test/figma/paint.test.js

```diff
@@ -71,6 +71,6 @@ describe('paint', () => {
         {color: {r: 0.875, g: 0.40833336114883423, b: 0.14583337306976318, a: 1}, position: 0.7677738070487976},
       ],
     }),
-  ).to.equal('linear-gradient(-58.32deg, #f2e03a 21.02%, #df6825 76.44%)');
+  ).to.equal('linear-gradient(-58.32deg, #f2e03a 11.87%, #df6825 76.78%)');
```

Comment:
I "fixed" this test, but it was same broken in `main` branch.
Seems to me it's a bug in the `linearGradient` function, which should also take `gradientHandlePositions` into account when computing stops.

Reply:
Yes this was a bit of some unfinished work I need to wrap up. Thanks!